doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -287,6 +287,24 @@ an opaque way. Nonetheless, the types described below are used to convey an spec
 }
 ```
 
+## Virtual attributes
+
+There are entity properties that are not directly manipulable by NGSIv2 clients, but that can be rendered
```

Comment:
manipulable ... Not sure that is an English word ...
I would use **modifiable** instead.